cmd/sops/main.go

```diff
@@ -695,15 +695,18 @@ func keyGroups(c *cli.Context, file string) ([]sops.KeyGroup, error) {
 	return []sops.KeyGroup{group}, nil
 }
 
+// loadConfig will look for an existing config file, either provided through the command line, or using config.FindConfigFile.
+// Since a config file is not required, a return value of nil, nil is possible
```

Comment:
Since a config file is not required, this function does not error when one is not found, and instead returns a nil config pointer

Reply:
Ok. Changing.